docs/91_version_history.rst

```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
+       - The tab view of tabular data in the user interface has been updated to show the date as well as the time related to readings.
 
 
-- **Plugins**
+- **Services & Plugins**
 
-    - New Features:
+    - Features
 
-       - The S2OPCUA south plugin has been updated to resolve an issue with duplicate browse names causing data from two OPCUA variables being stored in the same Fledge datapoint. The plugin has also been updated to give more option for how the assets are structured. The option of a single asset for all datapoints and an asset put OPCUA object have been added. It is also possible to use the OPCUA object name as the prefix for asset names in the case of a single variable per asset as well as the current option of a fix prefix for the browse name of the variable.
        - The asset filter has been updated to support the ability to map datapoint names for an asset.
-       - The OMF north plugin has been updated to allow support for OMF 1.2 features. This allows for better control of types within OMF resulting in the OMF plugin now dealign more cleanly with assets with different datapoint in different readings. Any assets that are already being sent to an OMF endpoint will continue to use the previous type mechanism. A number of new OMF hints are also supported.
-       - A new north plugin, foglamp-north-mqtt-scritped, has been created to send data north using MQTT. It provides a mechanism to use a Python script to format the payload of the MQTT messages. The plugin also supports the flow of control messages that trigger writes and operations from MQTT message at the plugin.
+       - The OMF north plugin has been updated to allow support for OMF 1.2 features. This allows for better control of types within OMF resulting in the OMF plugin now dealign more cleanly with assets with different datapoints in different readings. Any assets that are already being sent to an OMF endpoint will continue to use the previous type mechanism. A number of new OMF hints are also supported.
+       - The S2OPCUA south plugin has been updated to resolve an issue with duplicate browse names causing data from two OPC UA variables being stored in the same Fledge datapoint. The plugin has also been updated to give more options for how the assets are structured. The option of a single asset for all datapoints and an asset put OPC UA object have been added. It is also possible to use the OPC UA object name as the prefix for asset names in the case of a single variable per asset as well as the current option of a fix prefix for the browse name of the variable.
+       - The S2OPCUA south plugin has been updated to allow the timestamp for readings to be taken from the OPC UA server itself rather than the time that it was received by Fledge.
 
 
-    - Bug Fix:
+    - Bug Fixes
 
+       - A number of bug fixes have been made to the North OPC UA Client plugin to improve the logging and responsiveness of the plugin.
```

Comment:
Fine; though packages are being released first time?

Reply:
So, this sanitized description is not required and should be removed.